File: hp-server/src/main/java/net/hserver/hplite/domian/entity/UserDeviceEntity.java
package net.hserver.hplite.domian.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("user_device")
public class UserDeviceEntity {

    @TableId(type = IdType.AUTO)
    private Integer id;

    /**
     * 设备key
     */
    private String deviceKey;

    /**
     * 描述
     */
    private String remarks;

}

File: hp-server-golang/hp-server-lib/entity/user_device_entity.go
package entity

type UserDeviceEntity struct {
	Id *int `json:"id" gorm:"primaryKey;autoIncrement"`

	/**
	 * 设备key
	 */
	DeviceKey string `json:"deviceKey"`

	/**
	 * 描述
	 */
	Remarks string `json:"remarks"`

	/**
	 * 用户ID
	 */
	UserId *int `json:"userId"`
}

func (UserDeviceEntity) TableName() string {
	return "user_device"
}
